python/GafferArnold/ArnoldRender.py

```diff
@@ -64,6 +64,13 @@ def __init__( self, name="ArnoldRender" ) :
 		)
 
 		self.addChild(
+			Gaffer.StringPlug(
+				"command",
+				defaultValue = "kick -dp -dw",
```

Comment:
I feel like the `-dp -dw` part would be best left inside the node, and the documentation also seems to imply that it would be. Any particular reason to expose it?